### PR TITLE
Updates to the Edit (Experimental) Pattern

### PR DESCRIPTION
**Pull request by chloepoulterdesign**
Closes no issues

This PR updates the images of the full-page edit pattern as they were out of date. 

#### Changelog

**New** 
- Section for low-impact full-page edit, including 1 image

**Changed**
- 2 images in high-impact full-page edit

**Removed**
- none

### CODE REVIEW

**Comment by connor-leech** [comment]
Looks like you're getting a merge conflict but I'm not entirely sure why. You could try renaming the names of the new images (01, 04, 06, 07) and see if that resolves it

File: src/pages/experimental/edit-pattern/index.mdx [hunk 1]
```diff
@@ -73,6 +72,22 @@ If an object was created in a side panel, its editable fields can be made availa
 </Column>
 </Row>
 
+### Full page
+
+When an object is created in a full page flow and the majority of fields become editable, a full page edit flow should be used. This is made possible by also including a configuration/properties view.
```
Comment (connor-leech) [inline src/pages/experimental/edit-pattern/index.mdx]:
```suggestion
When an object is created in a full-page flow and the majority of fields become editable, a full-page edit flow should be used. This is made possible by including a configuration/properties view.
```

**Comment by chloepoulterdesign** [comment]
for the life of me @connor-leech I can't figure out these conflicts! I've taken everything off, put it all back again, changed names. 
Unless you've got any more suggestions, I might delete the branch and/or my fork and start again.

**Comment by vpicone** [comment]
@chloepoulterdesign sorry this fell through the cracks, the merge conflict was on us. Are you good to merge?

**Comment by chloepoulterdesign** [comment]
Yes please @vpicone - maybe @connor-leech or @vikkipaterson can quickly review and approve if needed.

**Pull request merged**